llm_studio/src/models/text_causal_language_modeling_model.py

```diff
@@ -122,6 +110,39 @@ def __init__(self, cfg: Any):
         if self.cfg.prediction.metric == "Perplexity":
             self.perplexity = Perplexity(self.cfg, reduce=False)
 
+    def prepare_lora(self):
+        target_modules = (
+            self.cfg.training.lora_target_modules.split(",")
```

Comment:
Can we add a `strip` here for extra spaces in the list?

Reply:
Good idea, the code now applies the strip method to both `self.cfg.training.lora_target_modules` as well as each target module.